auth.txt

```diff
@@ -0,0 +1 @@
+ac849baa-3cb1-40ef-a464-92460fa47617
```

Comment:
Ooh lets add this `auth.txt` file to the `.gitignore` so that git doesn't pick it up 
(and you might need to run `git rm --cached` to make git 'forget' about this file)

So if you add `cli/auth.txt` as its own line to the `.gitignore` file it will stop picking up changes to this file so each time we login/out we dont get a git diff!